starlette/templating.py

```diff
@@ -142,17 +142,16 @@ def get_template(self, name: str) -> "jinja2.Template":
 
     def TemplateResponse(
         self,
+        request: Request,
```

Comment:
Any reason why this is not an optional argument so it can be released without breaking?

Reply:
This was discussed on https://github.com/encode/starlette/pull/2191/.

The diff here is actually incomplete.